client/src/components/QuestionsAndAnswers/AddModal.jsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
```

Comment:
could import React, {useState} from 'react'; in order to not have to repeat React.useState